servicetalk-dependencies/build.gradle

```diff
@@ -26,28 +26,29 @@ javaPlatform {
 dependencies {
   api platform(project(":servicetalk-bom"))
   api platform("io.netty:netty-bom:${nettyVersion}")
-  api platform("com.fasterxml.jackson:jackson-bom:${jacksonVersion}")
+  api platform("com.fasterxml.jackson:jackson-bom:2.13.2.20220328")
```

Comment:
Thanks for researching this!

We use only one module (`jackson-databind`) from `com.fasterxml.jackson`. We can skip this bom if it creates more pain instead of value.

For us it will be easier to track only `jackson-databind` updates instead of the `jackson-bom`. With the bom, we will have to always keep the bom version the latest, even if `jackson-databind` remains the same version when they hot fix some other module. 

Protobuf and log4j don't have the same issue bcz they always keep version number consistent across modules.

Reply:
Agreed. I will remove the Jackson-bom